workspaces/server/src/db/schema/app.ts

```diff
@@ -93,6 +94,26 @@ export const userRolesRelations = relations(userRoles, ({ one }) => ({
 	}),
 }));
 
+export const socialLinks = sqliteTable(
+	"social_links",
+	{
+		userId: text("user_id")
+			.references(() => users.id)
+			.notNull(),
+		url: text("url").notNull(),
+	},
+	(table) => ({
+		pk: primaryKey({ columns: [table.userId, table.url] }),
+	})
+);
```

Comment:
たぶん primaryKey は url だけでいい気がする (ここに載せられるべきソーシャルアカウントは共有されていないはずなので)
でもそうすると user -> link 検索遅くなっちゃうので userId に index を張りましょう

つまり、

```suggestion
export const socialLinks = sqliteTable(
	"social_links",
	{
		userId: text("user_id")
			.references(() => users.id)
			.notNull()
			.primaryKey(),
		url: text("url").notNull(),
	},
	(table) => [
		index("userid_idx").on(table.userId)
	]
);
```
たぶんこんなかんじ